examples/basic/add.go

```diff
@@ -44,27 +48,35 @@ func main() {
 	}
 
 	// Read two args to add.
-	x, y := readTwoArgs()
+	x, y, err := readTwoArgs(flag.Arg(0), flag.Arg(1))
+	if err != nil {
+		log.Fatalf("failed to read arguments: %v", err)
```

Comment:
we've been asked to put Panicln in the examples, as while they are in the form of a CLI, they are tinkering areas. panics show you the line that failed.

Reply:
Not a problem, I am happy to restore the panic. I just thought that running an example binary with no arguments and getting a panic seemed a little unfriendly.

Now that there are checks for empty positional arguments, it's more clear where things went wrong.